core/src/main/clojure/xtdb/table_catalog_test.clj

```diff
@@ -41,8 +42,8 @@
         (xt/execute-tx node [[:put-docs :foo {:xt/id 2}]])
         (tu/finish-block! node)
 
-        (t/is (= [(os/->StoredObject "tables/public$foo/blocks/b00.binpb" 286)
-                  (os/->StoredObject "tables/public$foo/blocks/b01.binpb" 415)]
+        (t/is (= [(os/->StoredObject "tables/public$foo/blocks/b00.binpb" 4392)
+                  (os/->StoredObject "tables/public$foo/blocks/b01.binpb" 8627)]
```

Comment:
A 10x increase in size. Might be better to only add them (HLL's) at L1.

Reply:
do we get any choice over the sizing parameters here, if you're concerned about the size?